.eslintrc.js

```diff
@@ -46,5 +46,13 @@ module.exports = {
                 "no-var": ["off"],
             }
         },
+        {
+            "files": [
+                "src/js/**/*.js"
+            ],
+            "rules": {
+                "comma-dangle": ["error", "only-multiline"]
```

Comment:
Hmmm, could we add this with the other rules, above? Or is there a reason you created it as a separate section?

Reply:
@cdrini in the PR it was mentioned only to JS in src/
If you want I can add this rule with other rules

Reply:
Let's try adding it to everything :+1: